NuGet.Config

```diff
@@ -3,13 +3,14 @@
   <packageRestore>
     <add key="enabled" value="True" />
   </packageRestore>
+  <!-- the commented  nuget feeds had failing mirrors (only non windows). Temporarily disabling them -->
```

Comment:
@rainersigwald 

That is a very good question. The weird thing is that restoring from those feeds only fails on non windows. That's why our xplat CI is broken on osx right now, but passes on windows.

If I remove those offending feeds, all the packages get "magically" downloaded from some other feeds, which is a happy coincidence. I would say to leave them commented for a few months, and if the happy coincidence does not burst into an unhappy failed restore, completely remove them.